test/sass/scss/scss_test.rb

```diff
@@ -4221,4 +4221,70 @@ def test_crlf
 SCSS
   end
 
+  def test_dynamic_include
```

Comment:
When writing Ruby tests, I've been trying to create new files for new features. I'm sick of having just a few gargantuan test files that we have to scroll through to find anything.

That said, can we just add tests only to sass-spec? I don't think it's necessary to have Ruby tests for something that's tested there.